tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -646,7 +642,40 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
     // and B are later exchanged, and B is transposed here instead.
     // TODO(philipphack): Remove once cuBLASLt supports the NN configuration.
     TF_ASSIGN_OR_RETURN(bool is_col_major,
-                        OutputIsColumnMajor(instr, gemm_backend_config));
+                        MatrixIsColumnMajor(instr, gemm_backend_config, "d"));
```

Comment:
`MatrixIsColumnMajor` doesn't exist. Did you forget to commit that change?

Reply:
Yes. Just added.